### DISCUSSION TITLE
Welcome to Chatbot-Using-OpenAI-API Discussions!

### DISCUSSION THREAD

**Discussion by iSiddharth20** [iSiddharth20/Chatbot-Using-OpenAI-API#1] [Announcements]
I'm going to keep this simple.

I hope you like the Codebase and the Idea behind it. Feel free to use the Code for Research and Innovation.

If you find anything that can be optimized, any functionality that could be of use: Feel free to send a Pull Request with Appropriate Comments.

**Comment by LokeshYarramallu** [comment]
hello sir

**Reply by iSiddharth20** [reply]
Hello Lokesh,
Call me Sid!
